### PR TITLE
Possible fix for Unconscious creatures.

### PR DESCRIPTION
**Pull request by Schizz**
Pushes any Unconscious creature away if gets hit.
**Haven't tested!**

### CODE REVIEW

File: src/config_magic.c [hunk 1]
```diff
@@ -829,6 +831,21 @@ TbBool parse_magic_shot_blocks(char *buf, long len, const char *config_textname,
                 break;
             }
           }
+	  case 9: // PUSHONHIT
+		  if (get_conf_parameter_single(buf, &pos, len, word_buf, sizeof(word_buf)) > 0)
+		  {
+			  k = atoi(word_buf);
+			  shotst->old->push_on_hit = k;
+			  n++;
+			  CONFWRNLOG("hnus read \"%s\" parameter in [%s] block of %s file. --->%d",
```
Comment (Loobinex) [inline src/config_magic.c]: This log message we removed.

File: src/config_magic.c [hunk 2]
```diff
@@ -70,6 +70,7 @@ const struct NamedCommand magic_shot_commands[] = {
   {"AREADAMAGE",      6},
   {"SPEED",           7},
   {"PROPERTIES",      8},
+  {"PUSHONHIT",       9},
```
Comment (Loobinex) [inline src/config_magic.c]: I don't know why all the other changes we already committed yesterday are listed again. I checked the main branch, and they are already in.

Comment (Schizz) [reply src/config_magic.c]: weird

**Comment by Loobinex** [comment]
It also still gives code conflicts because something went wrong. Is it an idea to close this pull request, for you to sync with the main branch and then recommit the changes you made and make a new pull request?

**Comment by Loobinex** [comment]
That all being said, I tested it and it works great. If you want to see for yourself, look at the test-map I made:
[map00213.zip](https://github.com/Loobinex/keeperfx-unofficial/files/1736528/map00213.zip)

1) Turn imprisonment on
2) Have a few trolls attack blue
3) Sell the guard-post which enables the create-imp spell

In the original you'll quickly see imps knocked out on the heart itself. With this new version the imps get pushed away about a single tile. They are still on the heart, but no longer in the way.

I'd be happy to merge, when possible. (Conflicts prevent merging right now.)
@spartahawk, are you happy to merge as well?

**Comment by Schizz** [comment]
ok, I am gonna close this request and make the sync soon as possible. 
Do you wish to push the unconscious units little bit more in the next version?

**Comment by Loobinex** [comment]
No, I think the distance is absolutely perfect. Far enough to be out of the way, not so far that it looks silly.

**Pull request closed without merging**